app/src/main/java/org/schabi/newpipe/player/notification/NotificationUtil.java

```diff
@@ -104,9 +107,11 @@ private synchronized NotificationCompat.Builder createNotification() {
             compactSlots[i] = compactSlotList.get(i);
         }
 
-        builder.setStyle(new androidx.media.app.NotificationCompat.MediaStyle()
-                    .setMediaSession(player.getMediaSessionManager().getSessionToken())
-                    .setShowActionsInCompactView(compactSlots))
+        final MediaStyle mediaStyle = new MediaStyle().setShowActionsInCompactView(compactSlots);
+        player.UIs().get(MediaSessionPlayerUi.class).flatMap(MediaSessionPlayerUi::getSessionToken)
+                        .ifPresent(mediaStyle::setMediaSession);
```

Comment:
I would write it in this style. Easier readable in my opinion since the relevant calls are structure under each other.
```suggestion
        player.UIs()
            .get(MediaSessionPlayerUi.class)
            .flatMap(MediaSessionPlayerUi::getSessionToken)
            .ifPresent(mediaStyle::setMediaSession);
```